src/cartridges/app_adyen_SFRA/cartridge/client/default/js/googlePayExpressCommon.js

```diff
@@ -39,15 +104,70 @@ async function init() {
       const googlePayButtonConfig = {
         showPayButton: true,
         buttonType: 'buy',
+        environment: window.environment,
+        emailRequired: true,
+        shippingAddressRequired: true,
+        shippingOptionRequired: true,
+        shippingAddressParameters: {
+          phoneNumberRequired: true,
+        },
+        billingAddressRequired: true,
+        billingAddressParameters: {
+          format: 'FULL',
+          phoneNumberRequired: true,
+        },
+        gatewayMerchantId: window.merchantAccount,
         configuration: googlePayConfig,
-        returnUrl: window.returnUrl,
-        isExpress: true,
+        callbackIntents: ['SHIPPING_ADDRESS', 'SHIPPING_OPTION'],
+        amount: JSON.parse(window.basketAmount),
+        onAuthorized: async (data) => {
+          const componentData = googlePayButton.data;
+          const stateData = {
+            paymentMethod: componentData.paymentMethod,
+            paymentType: 'express',
+          };
+          const customer = formatCustomerObject(data);
+          paymentFromComponent({ ...stateData, customer });
+        },
+        onSubmit: async () => {},
+        paymentDataCallbacks: {
+          onPaymentDataChanged() {
+            return new Promise((resolve) => {
```

Comment:
can you add a comment here with "todo" saying this is just a mock and needs to be replaced with actual implementation.

Reply:
Added!